docs/how-to/llms/local.md

```diff
@@ -0,0 +1,66 @@
+# How-To: Use local LLMs
+
+db-ally includes a ready-to-use implementation for local LLMs called [`LocalLLM`](../../reference/llms/local.md#dbally.llms.local.LocalLLM), which leverages the Hugging Face Transformers library to provide access to various LLMs available on Hugging Face.
+
+## Basic Usage
+
+Install the required dependencies for using local LLMs.
+
+```bash
+pip install dbally[localllm]
+```
+
+Integrate db-ally with your Local LLM
+
+First, set up your environment to use a Hugging Face model.
+
+```python
+
+import os
+from dbally.llms.localllm import LocalLLM
+
+os.environ["HUGGINGFACE_API_KEY"] = "your-api-key"
+
+llm = LocalLLM(model_name="local/meta-llama/Meta-Llama-3-8B-Instruct")
```

Comment:
I believe this `local` prefix was intended only for benchmarking purposes, here it's obvious that it's a local LLM, because of the class.